cmd/docker/docker.go

```diff
@@ -30,7 +30,7 @@ import (
 
 func main() {
 	err := dockerMain(context.Background())
-	if err != nil && !errdefs.IsCancelled(err) {
+	if err != nil && !errdefs.IsCancelled(err) && !errdefs.IsContext(err) {
```

Comment:
Why the `!errdefs.IsContext(err)`? IMO I'd like to see a message telling me what happened if a request timed out or some such.

Reply:
true, right now I don't want to print any exit status or error message when the context is cancelled. I think it might be possible to determine if this is a sigint/sigterm vs a timeout for example.